src/likelihoodratiotest.jl

```diff
@@ -6,17 +6,20 @@ Results of MixedModels.likelihoodratiotest
 ## Fields
 * `formulas`: Vector of model formulae
 * `models`: NamedTuple of the `dof` and `deviance` of the models
-* `tests`: NamedTuple of the sequential `dofdiff`, `deviancediff`, and resulting `pvalues`
+* `tests`: NamedTuple of the sequential `dofdiff`, `deviancediff`,
+           and resulting `pvalues`
 
 ## Properties
-* `deviance`
+* `deviance` : note that this is actually -2 log likelihood for linear models
+               (i.e. without the additive constant for a saturated model)
```

Comment:
```suggestion
               (i.e. without subtracting the constant for a saturated model)
```
To me, an "additive constant" sounds like it is going to be added to the value, not subtracted from the value.